changelog.d/251.misc

```diff
@@ -0,0 +1 @@
+Addresses #230, refactors sygnal to use fcm as app type rather than gcm, and warns in documents about the future depreciation of gcm as an app type.
```

Comment:
```suggestion
Addresses #230, refactors sygnal to use fcm as app type rather than gcm, and warns in documents about the future deprecation of gcm as an app type.
```